index.html

```diff
@@ -160,6 +160,7 @@ <h3>Group repositories</h3>
       <ul>
         <li><a href="https://github.com/w3c/ttwg/pulse">TTWG business</a></li>
         <li><a href="https://github.com/w3c/tt-reqs/pulse">TTWG future requirements</a></li>
+        <li><a href="https://github.com/w3c/charter-timed-text/pulse">Draft charter for the Timed Text Working Group</a></li>
```

Comment:
I don't think we would normally advertise a repo whose only purpose is for changing the charter, which happens only every couple of years or so.

Reply:
removed